src/frontend/src/pages/TeamsPage/TeamMembersPageBlock.tsx

```diff
@@ -0,0 +1,129 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Autocomplete, Button, Grid, IconButton, TextField, Typography, useTheme } from '@mui/material';
+import { useState } from 'react';
+import { useAuth } from '../../hooks/auth.hooks';
+import { useAllUsers } from '../../hooks/users.hooks';
+import { useSetTeamMembers } from '../../hooks/teams.hooks';
+import { Team, User } from 'shared';
+import { fullNamePipe } from '../../utils/pipes';
+import { Edit } from '@mui/icons-material';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import PageBlock from '../../layouts/PageBlock';
+
+interface TeamMembersViewProps {
+  team: Team;
+}
+
+const TeamMembersView: React.FC<TeamMembersViewProps> = ({ team }) => {
```

Comment:
Can you rename the file to match the component name

Reply:
I renamed the component to the file name since anthony asked me to change the file name previously.

Reply:
what reid is saying is that the name of the constant and what it's exported as here is not the same as the file name